script/eslint.ts

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env ts-node
+
+import ChildProcess = require('child_process')
+import Path = require('path')
+
+const ESLINT_ARGS = [
+  '--cache',
+  '--rulesdir=./eslint-rules',
+  './{script,eslint-rules}/**/*.{j,t}s?(x)',
+  './tslint-rules/**/*.ts',
+  './app/{src,typings,test}/**/*.{j,t}s?(x)',
+  ...process.argv.slice(2),
+]
+const opts = {
+  stdio: 'inherit',
+}
+
+if (process.env.CI) {
+  ChildProcess.spawn(
+    Path.resolve(__dirname, '..', 'node_modules', '.bin', 'eslint'),
```

Comment:
Alright, this is the root cause of the AppVeyor problem.

Yes, the `eslint` command exists here when you unpack it on Windows.

No, these shims are not designed to be platform agnostic - `npm` will invoke the extension-less command on UNIX-based platforms, but on Windows `npm` will invoke the `.cmd` extension.

Don't try to be clever here with running the scripts in a bash file as a workaround. Just conditionalize it:

```ts
  const file = process.platform === 'win32'
    ? 'eslint.cmd'
    : 'eslint'

  const path = Path.resolve(__dirname, '..', 'node_modules', '.bin', file)
```